index.php

```diff
@@ -2,6 +2,39 @@
 $is_auth = rand(0, 1);
 
 $user_name = 'Алексей'; // укажите здесь ваше имя
+
+$articles = [
+    ['title' => 'Цитата',
```

Comment:
Тут следует писать с новой строчки
```suggestion
    [
        'title' => 'Цитата',
```